index.bs

```diff
@@ -490,7 +490,7 @@ The header's ABNF is:
 
 To <dfn abstract-op>return the `Sec-CH-UA` value for a request</dfn>, perform the following steps:
 
-1. Let |version type| be the type [=user agent/significant version=].
+1. Let |version type| be "significant version".
```

Comment:
Not critical, but you could skip the variable entirely and pass the value along to the algorithm.

Reply:
👍